Gordon360/Models/ViewModels/RecIM/SchedulerViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Gordon360.Models.CCT;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class SchedulerViewModel
+    {
+        public DateTime StartTime { get; set; }
+        public DateTime EndTime { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
```

Comment:
Is there a reason that the dates and times are separate? Could they be combined?

Reply:
Okay, cool. I was just checking then.